src/main/java/org/llorllale/cactoos/matchers/InputHasContent.java

```diff
@@ -40,6 +40,11 @@
  * @author Yegor Bugayenko (yegor256@gmail.com)
  * @version $Id$
  * @since 0.11
+ * @todo #7:30min Tests for all matchers should be added and in particular
```

Comment:
@victornoel tests for all matchers is being handled starting in #1

Reply:
@llorllale yes, I noticed that the other day, but then I thought about leaving this todo because it concerns mainly `describeMismatchSafely` in the end.
I propose to reformulate it without the "tests for all matchers should be added", what do you think?

Reply:
@victornoel yes, please just leave out the part about "tests for all matchers should be added". The rest of the puzzle is fine